src/north/north-connector.js

```diff
@@ -44,14 +47,12 @@ class NorthConnector {
     this.logParameters = configuration.logParameters
     this.caching = configuration.caching
     this.subscribedTo = configuration.subscribedTo
+    this.proxies = proxies
```

Comment:
can't we simplify a bit this, we should provide an array of proxies and read from config the selected proxy and directly create a proxyAgent as sole internal variable. Does this make sense?

Reply:
I saw several "problems" with that :
- the proxy to use is set in the connector specific settings, not in the global north settings, so I prefer not to retrieve it from the main north constructor as it is now. That is why the proxySettings are set in AWS3, OIA and OIC connectors, not in NorthConnector constructor
- The ProxyAgent constructor is async and must be called in the start method. One possibility was to add the proxy list in the start method arguments, but it is used by only three connectors.

Overall, I suggest to make a separate dev issue for that.